### DISCUSSION TITLE
适兕为什么选择开源？——致所有好奇的朋友 - 开源之道

### DISCUSSION THREAD

**Discussion by giscus** [OCselected/ocselected.github.io#24] [Announcements]
# 适兕为什么选择开源？——致所有好奇的朋友 - 开源之道

https://opensourceway.community/posts/the_way_of_open_source/kuosi-need-answer-questions/

**Comment by ZoomQuiet** [comment]
#是也乎,(￣▽￣)

将自己变成一粒合格的思想转换器, 
坚实的嵌入到当下中国社会里,

没毛病, 在多数人难以静心读书, 同步世界思想者们的思考时, 
有人愿意先为自己, 顺便为全中国程序猿读书, 认真观察/思考/分析/分享/...开源世界本身,
这很有意义,

其实, 这种行为, 就是古代标准士大夫, 代天巡检的工作,
只是以往有专门职位供养,
现在要自己找变现渠道而已...

但是, 可以认真结合以往开发经历, 重新思考基本问题:

- 开源原本是如何?
- 开源应该能如何?
- 开源最好还应怎样?

足以给所有涉及其中的人们, 更多思考选项.

值得开始私人开源研究史的记要:

- 哪天, 看了什么书
- 和谁, 聊了什么
- 在哪儿, 写下了什么
- ...以及每个行为,分别花了多少时间

慢慢积累起来, 以后统计成图表, 就可以清晰的看到, 一个普通人为中国开源思想史框架的确立投入了多少成本,
以社会平均工资计算, 这些成本就是其它国家无法超越的门槛了...

参考: [The Personal Analytics of My Life—Stephen Wolfram Writings](https://writings.stephenwolfram.com/2012/03/the-personal-analytics-of-my-life/)
